cmd/dispatcher/main.go

```diff
@@ -141,9 +141,9 @@ func (env *envConfig) setupRabbitMQ(ctx context.Context) {
 	}
 
 	err = env.channel.Qos(
-		env.Parallelism, // prefetch count
-		0,               // prefetch size
-		false,           // global
+		100,   // prefetch count
```

Comment:
Why not 1000? Are we sure that the 100 is a good value?

Reply:
Since it's a single consumer, a higher prefetch value would generally increase message delivery. On the flip side, memory usage will get impacted with a higher prefetch count. I chose 100 as it's something that's been tested in our performance tests. There isn't an ideal number that would work for all scenarios a user can come across.

Reply:
Kk maybe for the future we could start playing with those numbers, but approving :)